.bundle/config

```diff
@@ -1,2 +1,3 @@
 ---
 BUNDLE_BUILD__TINY_TDS: "--with-freetds-include=/opt/homebrew/include --with-freetds-lib=/opt/homebrew/lib"
+BUNDLE_PATH: "vendor/bundle"
```

Comment:
I don't think this should be here?

Reply:
If you need or want your bundle in the app directory, you can add ~/.bundle/config and put it there I think.

Reply:
Cool, just removed that 👍